nastaveni/hlasky/nastaveni-hlasky-subst.php

```diff
@@ -25,7 +25,9 @@
     'rychloregMail'                         =>
         'Ahoj,
 
-děkujeme, že se letos účastníš GameConu. Kliknutím na odkaz níže potvrdíš registraci na web a můžeš si nastavit přezdívku a heslo, pokud chceš používat web a třeba přijet i příští rok. (V případě pozdější registrace na web by bylo nutné nechat si vygenerovat heslo znova)
+děkujeme, že se letos účastníš GameConu! Kliknutím na odkaz níže potvrdíš registraci na web, kde si nastavíš přezdívku a heslo tak, ať můžeš používat web a přijet třeba i příští rok. V případě pozdější registrace na web by bylo nutné nechat si vygenerovat heslo znovu.
+
+
```

Comment:
Once again, TATO HLÁŠKA SE NIKDE NEPOUŽÍVÁ PROTOŽE NESBÍRÁME MAILY U RYCHLOREGISTRACÍ

Reply:
Nemam ponětí jak to funguje. Napsala to Danča, tak jsem to nahradil. Jestli se to používá nebo ne, nikomu neuškodí jestli to tam bude.